website/content/en/scripts/step05-controller-iam.sh

```diff
@@ -0,0 +1,66 @@
+echo "{
```

Comment:
Can we refer to an already created file rather than an in-line bash blurb? I can imagine someone might forget to update this file (and others) or may mess up the formatting, where it should be simple to check by eye in a separate file. 

For reference, I mean something similar to 
https://github.com/aws/karpenter/blob/main/website/content/en/v0.22.0/getting-started/getting-started-with-eksctl/cloudformation.yaml. 

Let me know if I'm missing something here.